packages/chakra-components/src/components/Election/Questions.tsx

```diff
@@ -410,10 +415,14 @@ const QuestionsTypeBadge = () => {
       return null
   }
 
+  const weighted = Number(election.census.weight) !== election.census.size && localize('question_types.weighted_voting')
+
   return (
     <chakra.div __css={styles.box}>
       <Tooltip label={tooltip} hasArrow sx={styles.tooltip} placement='auto'>
-        <chakra.label __css={styles.title}>{title}</chakra.label>
+        <chakra.label __css={styles.title}>
+          {title} {weighted}
```

Comment:
Concatenated strings should not be used on a localized application. Use a translation with interpolation instead.

Reply:
6ddfcbf fix it. 

But... @jpaulet could not be best if I just add a new "badge" for weighted voting? On this way the weighted voting badge can have its own tooltip and we can avoid to adapt every location for the badges with the weighted parameter (see the commit)